dogs.html

```diff
@@ -0,0 +1,148 @@
+   -->	
+	<h2>Seattle’s premiere dog adoption agency</h2>
+	<a href="#"><img src="images/cart-icon.svg" alt="checkout image">Checkout</a>
+</header>
+
+	<nav>
+		<a href="index.html">Home</a> /
+		<a href="dogs.html">Dogs</a> / 
+		<a href="blog.html">Blog</a>
+	</nav>
+
+<section>
+	<h1>Hare are all of our Dogs! </h1>
+	<div class="flex-container">
+		<div>
+		<img src="images/murphy-card.jpg" alt="a boy huges a dog image">
+		<dl>
+		<dt>Murphy</dt>
+		<dt>Cost to Adopt: $123.45</dt>
+		<dd>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</dd>
+		</dl>
+		</div>
+
+		<div>
+		<img src="images/poppy-card.jpg" alt="a boy huges a dog image">
+		<dl>
```

Comment:
I don't think this is a good use of dt and dl tags. I'd consider using more basic tags.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl